CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
   * Example updated to work with CocoaPods 1.0
   * Private headers are marked private in Podspec
   * Adds subspec for Health support
+* Improves Swift support by adding nullability annotations
```

Comment:
It's a new bullet point whereas the others are sub-points of one separate bullet point